lib/Environment.php

```diff
@@ -508,10 +514,9 @@ public static function slugify(string $text): string
 
     private function addMissingReferenceSectionError(string $section): void
     {
-        $this->errorManager->error(sprintf(
-            'Unknown reference section "%s"%s',
-            $section,
-            $this->getCurrentFileName() !== '' ? sprintf(' in "%s" ', $this->getCurrentFileName()) : ''
-        ));
+        $this->errorManager->addError(
+            sprintf('Unknown reference section "%s"', $section),
+            $this->getCurrentFileName(),
```

Comment:
This is not valid for old versions of PHP.
```suggestion
            $this->getCurrentFileName()
```

Reply:
Thank you. I could not find that syntax error.